tests/integration/pipelines/pipeline_container_no_input.yaml

```diff
@@ -0,0 +1,27 @@
+# PIPELINE DEFINITION
+# Name: v2-container-component-no-input
```

Comment:
I think this an artifact generated by running `tests/integration/pipelines/pipeline_container_no_input.py`?  Can we add a note here saying this is a generated file and it should not be edited manually?
```suggestion
# PIPELINE DEFINITION
# Name: v2-container-component-no-input
# Note: This file is auto-generated by running `pipeline_container_no_input.py`.  
#           Do not edit this file manually, edit and run the `.py`
```